### DISCUSSION TITLE
Location sharing; Noone

### DISCUSSION THREAD

**Discussion by TheAngryRogue** [smasherprog/EqTool#57] [General]
The change log makes it seem like the choice to NOT share my location has been removed permanently.  Is my assumption correct and if so why?

**Comment by neyacinu** [comment]
Definitely would like to share with nobody else.  Even better would be the ability to share with specific people.

**Comment by smasherprog** [comment]
If you set it to guild only, then only other guildmembers can see you. If you are NOT in a guild, and have guild only set, this will share your location with no one.

**Comment by smasherprog** [comment]
Also, location sharing is only to OTHERs in the Same zone as you unlike nparse which shares locations with the entire server. This is scoped to the zone you are in, so people in other zones cant see you, and visa virsa. So, worst case is a fellow guild member sees your location  when they are in the same zone as you.

**Comment by smasherprog** [comment]
Middle ground achieved!
